scripts/docker-helper-4.x/cht-docker-compose.sh

```diff
@@ -46,16 +46,20 @@ CHT_NETWORK=$projectName-cht-net
 EOL
 }
 
+get_staging_url_base() {
```

Comment:
Naming suggestion: what about `get_latest_release_url`? I feel like it needs the "latest release" bit to really convey what it's doing.